libs/api/domains/rights-portal/src/lib/api-domains-rights-portal.resolver.ts

```diff
@@ -0,0 +1,35 @@
+import { Query, Resolver } from '@nestjs/graphql'
+import { ApiScope } from '@island.is/auth/scopes'
+import { UseGuards } from '@nestjs/common'
+import { Audit } from '@island.is/nest/audit'
+import {
+  IdsUserGuard,
+  ScopesGuard,
+  Scopes,
+  CurrentUser,
+} from '@island.is/auth-nest-tools'
+import type { User } from '@island.is/auth-nest-tools'
+import { RightsPortalService } from './api-domains-rights-portal.service'
+import { Therapies } from './models/getTherapies.model'
+import { AidsAndNutrition } from './models/getAidsAndNutrition.model'
+
+@UseGuards(IdsUserGuard, ScopesGuard)
+@Resolver()
+@Audit({ namespace: '@island.is/api/rights-portal' })
+export class RightsPortalResolver {
+  constructor(private readonly rightsPortalService: RightsPortalService) {}
+
+  @Scopes(ApiScope.health)
+  @Query(() => [Therapies], { nullable: true })
+  @Audit()
+  getRightsPortalTherapies(@CurrentUser() user: User) {
+    return this.rightsPortalService.getTherapies(user)
+  }
```

Comment:
All resolvers and mutations should follow our [naming convention](https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions). E.g.

```suggestion
  @Scopes(ApiScope.health)
  @Query(() => [Therapies], { name: 'rightsPortalTherapies', nullable: true })
  @Audit()
  getRightsPortalTherapies(@CurrentUser() user: User) {
    return this.rightsPortalService.getTherapies(user)
  }
```